tools/get_latest_js_types.bat

```diff
@@ -0,0 +1,11 @@
+@echo off
+:: Get latest types file and plop it in scripts
+
+powershell Invoke-WebRequest https://github.com/panorama-languages-support/panorama-jsdoc-gen/releases/latest/download/types_momentum.zip -OutFile types.zip
```

Comment:
We're not momentum, I'm pretty sure this generates types for p2ce too

Reply:
This isn't needed for us, it's inside sdk_tools